src/Umbraco.Cms.Api.Management/ViewModels/Segment/SegmentResponseModel.cs

```diff
@@ -0,0 +1,12 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Umbraco.Cms.Api.Management.ViewModels.Segment;
+
+public class SegmentResponseModel
+{
+    [Required]
+    public string Name { get; set; } = string.Empty;
+
+    [Required]
+    public string Alias { get; set; } = string.Empty;
```

Comment:
I think we can just use the keyword here instead 😁 
```suggestion
    public required string Name { get; set; } = string.Empty;

    public required string Alias { get; set; } = string.Empty;
```